### PR TITLE
Add implementation

### PR DESCRIPTION
**Pull request by morrisallison**
Initial implementation of the module.
Version, changelog, and publishing will go in another PR.

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
```
Comment (LionOps) [inline .jshintrc:1]: This should merely be 

```
{
  "extends": "./node_modules/jsgreat/es6/.jshintrc"
}
```

Comment (LionOps) [inline .jshintrc:1]: Also, jsgreat needs added to the dependencies

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  - '0.10'
+  - '0.12'
+  - stable
+
+install:
+  - npm install
+
+script:
+  - npm test
+
```
Comment (stefanor) [inline .travis.yml]: End text files with a newline.

File: .jshintrc (hunk 1 above)
Comment (morrisallison) [inline .jshintrc:1]: Thanks, I'm facepalming for just now seeing jsgreat was one of ours

File: package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/yola/ios-detector"
+  },
+  "bugs": {
+    "url": "https://github.com/yola/ios-detector/issues"
+  },
+  "devDependencies": {
+    "babel-preset-es2015": "6.3.*",
+    "babelify": "7.2.*",
```
Comment (LionOps) [inline package.json]: This is a dependency. It's needed by browserify.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    "chai": "3.4.*",
+    "jsgreat": "0.1.*",
```
Comment (LionOps) [inline package.json:35]: This might be needed as a dependency. Depends on whether anything that references it is packaged.

**Comment by morrisallison** [comment]
Quick note on `"files"` in package.json in case someone is wondering why I left them out

> Certain files are always included, regardless of settings:
> - package.json
> - README (and its variants)
> - CHANGELOG (and its variants)
> - LICENSE / LICENCE

https://docs.npmjs.com/files/package.json#files

File: tests/ios-detector-spec.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+} from './helpers/user-agents.js';
+
+const htmlEl = document.documentElement;
+const iosPlatforms = [
+  'iPad',
+  'iPhone',
+  'iPod'
+];
+
+chai.should();
```
Comment (LionOps) [inline tests/ios-detector-spec.js:24]: I am biased against `should` as it has [limitations](http://chaijs.com/guide/styles/) when it comes to certain types of tests. I'm aware that there are workarounds, but `expect` doesn't have those limitations.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "ios-detector",
+  "version": "0.0.0",
```
Comment (LionOps) [inline package.json:3]: Symver suggests starting at `0.1.0`. See first question in the [FAQ](http://semver.org/).

**Comment by LionOps** [comment]
:+1:

File: package.json (hunk 6 above)
Comment (morrisallison) [inline package.json:3]: Yup, that'll go in the next PR

**Pull request merged**